src/index.html

```diff
@@ -1,15 +1,192 @@
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST">
+      <fieldset class="set">
+        <legend>Personal information</legend>
+        <label class="form-field" for="surname">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="name">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            id="name"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="age">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="0"
+            max="300"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="birthdate">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+            id="birthdate"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="term-of-the-agreement">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            id="term-of-the-agreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="set">
+        <legend>Registration</legend>
+        <label class="form-field" for="email">
+          E-mail:
+          <input
+            name="email"
+            placeholder="email@example.com"
+            type="email"
+            id="email"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="password">
+          Password:
+          <input
+            name="password"
+            type="password"
+            id="password"
+            minlength="8"
+            maxlength="16"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="set">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label for="yes">
+            <input
+              name="cats"
+              type="radio"
+              id="yes"
+              value="y"
+            >
+                Yes
+          </label>
```

Comment:
```suggestion
              value="y"
            >
              Yes
          </label>
```
Too many indentations